src/Printer_CartridgeInfo.php

```diff
@@ -41,7 +41,7 @@ class Printer_CartridgeInfo extends CommonDBChild
 
     public static function getTypeName($nb = 0)
     {
-        return _x('Cartridge inventoried information', 'Cartridge inventoried information', $nb);
+        return _x('Cartridge inventoried information', 'Cartridge inventoried information');
```

Comment:
"information" is uncountable in english, but not in other languages. Also `_x` expect a context, there does not seem to be one there.
```suggestion
        return _n('Cartridge inventoried information', 'Cartridge inventoried information', $nb);
```

Reply:
Hello, this isn't possible because the ``_n`` function doesn't support context (``$ctx``). There's the ``_nx`` function that teorically does this but the ``.po`` file should be updated so the string **Cartridge inventoried information** would be set as plural on the file and work as expected. I'm not sure why there was a context set for this string, I believe it was made by mistake. I believe the fix you propose would work only by updating the ``.po`` file.

Reply:
`getTypeName()` methods usually returns a "pluralized" string. Context initial usage here was just wrong.

Of course the fix will require PO files to be updated; that's not a problem.